azerothcore/azerothcore-wotlk
# (DB/Translations) Alliance quests in Grizzly Hills

## Changes Proposed:
-  Add the texts of the tables `quest_request_items_locale` and `quest_offer_reward_locale`. As well as some `ObjectiveText` from the `quest_template_locale` table that are untranslated.

## Issues Addressed:

- Closes 

## List of translated quests

- [x] 11986 - The Damaged Journal
- [x] 11988 - The Runic Keystone
- [x] 11993 - The Runic Prophecies
- [x] 11998 - Softening the Blow
- [x] 12002 - Brothers in Battle
- [x] 12003 - Uncovering the Tunnels
- [x] 12010 - The Fate of Orlond
- [x] 12014 - Steady as a Rock?
- [x] 12027 - Mr. Floppy's Perilous Adventure
- [x] 12105 - Descent into Darkness
- [x] 12109 - Report to Gryan Stoutmantle... Again
- [x] 12128 - Check Up on Raegar
- [x] 12129 - The Perfect Plan
- [x] 12130 - Why Fabricate When You Can Appropriate?
- [x] 12131 - We Have the Power
- [x] 12138 - ... Or Maybe We Don't
- [x] 12153 - The Iron Thane and His Anvil
- [x] 12154 - Blackout
- [x] 12158 - Hollowstone Mine
- [x] 12159 - Souls at Unrest
- [x] 12160 - A Name from the Past
- [x] 12161 - Ruuna the Blind
- [x] 12180 - The Captive Prospectors
- [x] 12183 - Looking the Part
- [x] 12184 - Cultivating an Image
- [x] 12185 - Put on Your Best Face for Loken
- [x] 12210 - Troll Season!
- [x] 12212 - Replenishing the Storehouse
- [x] 12215 - Them or Us!
- [x] 12216 - Take Their Rear!
- [x] 12217 - Eagle Eyes
- [x] 12219 - The Failed World Tree
- [x] 12220 - A Dark Influence
- [x] 12222 - Secrets of the Flamebinders
- [x] 12223 - Thinning the Ranks
- [x] 12225 - Mmm... Amberseeds!
- [x] 12226 - Just Passing Through
- [x] 12227 - Doing Your Duty
- [x] 12244 - Shredder Repair
- [x] 12246 - A Possible Link
- [x] 12247 - Children of Ursoc
- [x] 12248 - Vordrassil's Sapling
- [x] 12249 - Ursoc, the Bear God
- [x] 12250 - Vordrassil's Seeds
- [x] 12255 - The Thane of Voldrune
- [x] 12268 - Pieces Parts
- [x] 12289 - Kick 'Em While They're Down
- [x] 12292 - Local Support
- [x] 12293 - Close the Deal
- [x] 12294 - A Tentative Pact
- [x] 12295 - An Exercise in Diplomacy
- [x] 12296 - Life or Death
- [x] 12299 - Northern Hospitality
- [x] 12300 - Test of Mettle
- [x] 12302 - Words of Warning
- [x] 12307 - Wolfsbane Root
- [x] 12308 - Escape from Silverbrook
- [x] 12310 - A Swift Response
- [x] 12314 - Down With Captain Zorna!
- [x] 12316 - Keep Them at Bay!
- [x] 12323 - Smoke 'Em Out
- [x] 12414 - Mounting Up
- [x] 12437 - Riding the Red Rocket
- [x] 12443 - Seeking Solvent
- [x] 12444 - Blackriver Skirmish
- [x] 12511 - The Hills Have Us
- [x] 12770 - Reallocating Resources

## SOURCE:


## Tests Performed:

- The tests were made inside the game, executing one by one the SQL queries, which I have them in separate files, but I put them together, all in one file, to make it easier to apply and test.
- At the time of executing the queries, both separately and as a whole, none of them presented conflicts.


## How to Test the Changes:


1. If you take missions from that area, in the alliance faction, you will notice, with the client in Spanish, that many of the texts mentioned are untranslated.
2. You apply the SQL, delete the game client cache folder, and enter to test the quests (of the alliance only for the moment). The others are not finished yet.

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
